main/src/mockit/internal/startup/MockingBridgeFields.java

```diff
@@ -12,38 +12,93 @@
 import mockit.internal.*;
 import mockit.internal.expectations.mocking.*;
 import mockit.internal.mockups.*;
+import mockit.internal.util.*;
```

Comment:
Actually I also think star-imports are bad, as I want to see from the imports instantly where a referenced class is coming from, even if I'm not within an IDE. But if I contribute to foreign codebases, I usually adjust to the code-style I find and JMockit uses star-import everywhere.